dashboard/client/src/api/functionsApi.js

```diff
@@ -60,6 +60,15 @@ class FunctionsApi {
         shortSha = 'unknown';
       }
 
+      let isPrivate = false;
```

Comment:
Can't this be written simply as? `let isPrivate = item.labels['com.openfaas.cloud.git-private-repo'] && item.labels['com.openfaas.cloud.git-private-repo'] === 'true'`